src/test/javascript/spec/component/overview/course-conversations/layout/conversation-header/conversation-header.component.spec.ts

```diff
@@ -35,8 +51,18 @@ examples.forEach((activeConversation) => {
                     MockComponent(GroupChatIconComponent),
                     MockComponent(FaIconComponent),
                     MockPipe(ArtemisTranslatePipe),
+                    RouterTestingModule.withRoutes([
+                        { path: 'courses/:courseId/lectures/:lectureId', component: CourseLectureDetailsComponent },
+                        { path: 'courses/:courseId/exercises/:lectureId', component: CourseExerciseDetailsComponent },
```

Comment:
Is this intentional to have :lectureId here?

Reply:
No ;)